scripts/compsets/exglobal/exglobal_fcst_nems.sh

```diff
@@ -1422,11 +1438,335 @@ runSeq::
+
+#
+# Minimin and maximum values of vertical coordinate (km)
+# Used if #levels > 0
+#
+output_grid_level_range: 90. 1000.
+
+#
+# List of vertical levels (km)
+# Used if #levels = 0
+#
+output_grid_level_values::
+   90.
+  150.
+  300.
+  400.
+  500.
+  600.
+  700.
+  800.
+::
+
+#
+# If true, this flag transforms the vertical coordinate (v),
+# assumed in km, to heights (h) relative to the average
+# Earth's radius (R=6,371.2 km), accordina to the formula:
+#   h = 1 + v/R
+#
+output_grid_level_relative: false
+EOF
+
+cat << EOF > $DATA/swio.ipe.rc
+#############################################
+####  SWIO Run-Time Configuration File  #####
+#############################################
+#
+#  Imported fields
+#
+import_fields::
+    temp_neutral
+    eastward_wind_neutral
+    northward_wind_neutral
+    upward_wind_neutral
+    O_Density
+    O2_Density
+    N2_Density
+    O_plus_density
+    H_plus_density
+    He_plus_density
+    N_plus_density
+    NO_plus_density
+    O2_plus_density
+    N2_plus_density
+    O_plus_2D_density
+    O_plus_2P_density
+    ion_temperature
+    electron_temperature
+    eastward_exb_velocity
+    northward_exb_velocity
+    upward_exb_velocity
+::
+
+#
+# Output file format.
+#
+# Choose from:
+# - hdf5
+# - pnetcdf or parallel-netcdf
+#
+output_format: pnetcdf
+
+#
+# Output file prefix.
+#
+# Output file names are created as:
+#   <prefix>.<datetime>.<suffix>
+# where:
+#   <prefix>   : is the string provided via
+#                output_file_prefix keyword
+#   <datetime> : inported field timestamp,
+#                formatted as: YYYYMMDD_hhmmss
+#   <suffix>   : determined from output format:
+#                'hd5' for hdf5, 'nc' for pnetcdf
+#
+output_file_prefix: ipe
+
+#
+# Output grid type.
+# 
+# Supported types are:
+# - latlon: 2D/3D regular lat/lon spherical grid
+# - none  : native (imported) grid/mesh
+#
+output_grid_type: latlon
+
+#
+# The following keywords are read only if grid
+# type is 'latlon'
+#
+# Output grid size.
+#
+# Provides the lat/lon grid resolution as
+#   #longitudes #latitudes [#levels]
+
+# If #levels is missing, a 2D lat/lon grid will be built
+#
+# If #levels > 0, the minimum and maximum value of the
+# vertical coordinate (km) will be read via the 
+# 'output_grid_level_range' keyword below.
+#
+# IF #levels <=0, the grid's vertical levels are read
+# from the 'output_grid_level_values' table (km)
+#
+output_grid_size: 90 91 183
+
+#
+# Minimin and maximum values of vertical coordinate (km)
+# Used if #levels > 0
+#
+output_grid_level_range: 90. 1000.
+
+#
+# List of vertical levels (km)
+# Used if #levels <= 0
+#
+output_grid_level_values::
+   90.
+  150.
+  300.
+  400.
+  500.
+  600.
+  700.
+  800.
+::
+
+#
+# If true, this flag transforms the vertical coordinate (v),
+# assumed in km, to heights (h) relative to the average
+# Earth's radius (R=6,371.2 km), accordina to the formula:
+#   h = 1 + v/R
+#
+output_grid_level_relative: false
+EOF
 
 else
 
```

Comment:
We should link the new output files from RUNDIR to ROTDIR; I can help with this if needed.

Reply:
Agreed. Please make all necessary changes.